client/src/redux/p2p/p2pActionCreators.js

```diff
@@ -72,8 +72,8 @@ export const p2pBroadcast = (data) => {
   });
 };
 
-export const p2pBroadcastWinnerId = (peerId) => {
-  p2pBroadcast({ winnerId: peerId });
+export const p2pBroadcastGameResult = (result) => {
+  p2pBroadcast({ result });
 };
```

Comment:
We must broadcast the peerId (or an array of peerIds), not the username(s), because the default usernames will differ for each peer.  Your 'Player 5' might be my 'Player 3', for example.

In other words, usernames must be resolved locally for each peer.